app/src/main/res/layout/user_profile_screen.xml

```diff
@@ -0,0 +1,180 @@
+            android:layout_marginRight="4dp"
+            android:elevation="9dp"
+            app:cardCornerRadius="24dp"
+            tools:targetApi="lollipop">
+            <androidx.appcompat.widget.AppCompatImageView
+                android:layout_width="match_parent"
+                android:layout_height="match_parent"
+                android:elevation="9dp"
+                android:contentDescription="@string/edit_icon_string"
+                android:src="@drawable/ic_editiconwithellipse"
+                tools:targetApi="lollipop" />
+
+        </androidx.cardview.widget.CardView>
+    </androidx.cardview.widget.CardView>
+
+    <androidx.cardview.widget.CardView
+        android:layout_width="300dp"
+        android:layout_height="84dp"
+        android:layout_gravity="center"
+        android:layout_marginTop="20dp"
+        app:cardElevation="0dp">
+
+        <androidx.appcompat.widget.LinearLayoutCompat
+            android:layout_width="260dp"
+            android:layout_height="wrap_content"
+            android:layout_gravity="center"
+            android:background="@drawable/user_profile_card_background"
+
+            android:orientation="horizontal"
+            android:padding="8dp"
+
+            tools:targetApi="lollipop">
+
+            <androidx.appcompat.widget.AppCompatImageView
+                android:layout_width="24dp"
+                android:layout_height="24dp"
+                android:layout_gravity="center_vertical"
+                android:contentDescription="@string/person_image_string"
+                android:src="@drawable/ic_writing" />
+
+            <androidx.appcompat.widget.AppCompatEditText
+                android:id="@+id/userAbout"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_gravity="center_vertical"
+                android:layout_marginStart="10dp"
+                android:layout_marginLeft="10dp"
+                android:text="@string/about_status_string"
+                android:textSize="20sp" />
+        </androidx.appcompat.widget.LinearLayoutCompat>
+        <androidx.cardview.widget.CardView
+            android:layout_width="35dp"
+            android:layout_height="35dp"
+            android:layout_gravity="end"
+            android:layout_marginEnd="4dp"
+            android:layout_marginRight="4dp"
+            android:elevation="9dp"
+            app:cardCornerRadius="24dp"
+            tools:targetApi="lollipop">
+            <androidx.appcompat.widget.AppCompatImageView
+                android:layout_width="match_parent"
+                android:layout_height="match_parent"
+                android:elevation="9dp"
+                android:contentDescription="@string/edit_icon_string"
+                android:src="@drawable/ic_editiconwithellipse"
+                tools:targetApi="lollipop" />
+
+        </androidx.cardview.widget.CardView>
+    </androidx.cardview.widget.CardView>
+
```

Comment:
remove this extra line

Reply:
There can be a blank line after a view is ended. but not in between its properties.

Reply:
done